### PR TITLE
[KO] Translation of Plug lesson

### PR DESCRIPTION
**Pull request by riseshia**
Related to: #201 

@Dalgona @devleoper @rijgndqw012
Sadly, there is no lesson I could take on basic lesson...that's why I translate this.
Please review this when you have time, it's fine after basic lesson trslation done :)

### CODE REVIEW

File: ko/lessons/specifics/plug.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+    assert conn.state == :sent
+    assert conn.status == 404
+  end
+end
+```
+
+## 사용 가능한 Plug
+
+많은 Plug는 어려운 설정 없이 사용할 수 있습니다. 목록은 [여기](https://github.com/elixir-lang/plug#available-plugs)의 Plug 문서에 있습니다.
```
Comment (rijgndqw012) [inline ko/lessons/specifics/plug.md]: > (maybe) 많은 Plug는 -> 많은 Plug들을 (Plugs: plural)
> (maybe) 에 있습니다 -> 에서 찾을 수 있습니다. 
meaning of found is omitted. but, '에 있습니다' is not bad.

File: ko/lessons/specifics/plug.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
+---
+layout: page
+title: Plug
+category: specifics
+order: 1
+lang: ko
+---
+
+Ruby를 잘 알고 계신다면 Plug는 이곳저곳에 Sinatra의 영향을 받은 Rack이라고 생각해도 좋습니다. Plug는 Web 애플리케이션을 위한 명세와 Web 서버를 위한 어댑터를 제공합니다. Elixir 코어의 일부가 아닌, Elixir의 공식 프로젝트입니다.
+
```
Comment (rijgndqw012) [inline ko/lessons/specifics/plug.md]: how about '여기저기서' instead of '이곳저곳에' ?
'이곳저곳에' feels like 'give effect to anywhere'.

Comment (riseshia) [inline ko/lessons/specifics/plug.md]: `여기저기서` feels as similar as `이곳저곳에`... how about `여러 부분에서`?

File: ko/lessons/specifics/plug.md (hunk 1 above)
Comment (riseshia) [inline ko/lessons/specifics/plug.md]: I think it is more clear! I will update, thanks :)

File: ko/lessons/specifics/plug.md (hunk 2 above)
Comment (rijgndqw012) [inline ko/lessons/specifics/plug.md]: I agree. `여러 부분에서` is also natural.

**Comment by rijgndqw012** [comment]
pretty nice :+1:

File: ko/lessons/specifics/plug.md (hunk 2 above)
Comment (riseshia) [inline ko/lessons/specifics/plug.md]: Let's take this. Thanks to your suggestion 👍

**Pull request merged**